docs/products/access/api-reference/custom-registration/custom-registration-v2.md

```diff
@@ -83,15 +83,16 @@ Endpoint: `POST /oauth/v2/custom-registration/{idp}/complete`
-| `hook_context_custom_params` | no                                  | A map of custom web hooks context parameters used in [Web Hooks](../../topics/integration-extension/hooks/index.md)                                                                                       |
-| `grant_type`                 | no                                  | Grant type that the Access Token will be bound with. <br/> If not specified, the default grant type will be used. <br/> Available values: `urn:onewelcome.com:oauth2:grant_type:stateless_authentication` |
+| Param                        | Required                                | Description                                                                                                                                                                                               |
+|------------------------------|-----------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `client_assertion_type`      | yes                                     | The type of the client assertion that is part of the request, currently the only supported value is `urn:ietf:params:oauth:client-assertion-type:jwt-bearer`                                              |
+| `client_assertion`           | yes                                     | Client assertion in the form of a JWT signed by the client's private key                                                                                                                                  |
+| `transaction_id`             | yes (`TWO_STEP`) otherwise optional     | Generated in **Init step**. For `TWO_STEP`, ensures same transaction                                                                                                                                      |
+| `data`                       | no                                      | Raw registration request data which will be provided to the Extension Engine                                                                                                                              |
+| `scope`                      | no                                      | An array of scopes. If none are specified the default scopes are granted.                                                                                                                                 |
+| `profile_id`                 | yes (dynamic client) otherwise optional | The profile ID of the user on the Onegini SDK, static clients can omit this                                                                                                                               |
```

Comment:
Are there any limitations in the length of this? And are you sure that it is optional for webclients?

Reply:
Profile id must be 6 characters long.
For web clients, it's ignored - not validated nor used (set to `null` in Access Token).

Reply:
I found this in backstage docs:
`The profile id parameter (profile_id) value will be generated on the SDK side, and it will
be a random 6 character long string.`

Reply:
Can you add the 6 chars limitation? And maybe mention mobile clients instead on dynamic clients?

Reply:
done